### PR TITLE
Go Client Read FMC Smart License

### PR DESCRIPTION
**Pull request by weilueluo**
https://jira-eng-rtp3.cisco.com/jira/browse/LH-67704

### CODE REVIEW

**Review by hazital — APPROVED** [review]
LGTM, can you pls add a test for the ReadSmartLicense?

File: client/device/cdfmc/readsmartlicense.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package cdfmc
```
Comment (siddhuwarrier) [inline client/device/cdfmc/readsmartlicense.go:1]: Please use underscores in the name

**Review by siddhuwarrier — CHANGES_REQUESTED** [review]
Tal's comment on tests + comment on file name, otherwise LGTM.

File: client/device/cdfmc/readsmartlicense.go (hunk 1 above)
Comment (weilueluo) [reply client/device/cdfmc/readsmartlicense.go:1]: hmm I think Golang's convention is to use short and concise name with no underscore or mixed caps: https://go.dev/blog/package-names

**Comment by weilueluo** [comment]
> LGTM, can you pls add a test for the ReadSmartLicense?

Done 🙂

File: client/device/cdfmc/readsmartlicense.go (hunk 1 above)
Comment (siddhuwarrier) [reply client/device/cdfmc/readsmartlicense.go:1]: Fair enough. TIL!

**Review by siddhuwarrier — APPROVED** [review]

**Pull request merged**